partialWaveFit/fitResult.cc

```diff
@@ -998,3 +1000,149 @@ fitResult::intensityErr(const string& waveNamePattern) const
 	const TMatrixT<double> intensityCov = jacobian * prodAmpCovJT;              //  1 x  1 matrix
 	return (double)normNmbEvents() * sqrt(intensityCov[0][0]);
 }
+
+
+void fitResult::stripCovMatrix() {
+	_fitParCovMatrix.ResizeTo(0,0);
+	_fitParCovMatrixIndices.clear();
+	_hasHessian = false;
+	_covMatrixValid = false;
+}
+
+
+void fitResult::stripIntegralMatrices() {
+	_normIntegral.resizeTo(0,0);
+	_acceptedNormIntegral.resizeTo(0,0);
+	_phaseSpaceIntegral.clear();
+	_normIntIndexMap.clear();
+}
+
+
+std::map<rpwa::multibinBoundariesType, std::list<rpwa::fitResult> > rpwa::getFitResultsFromFilesInMultibins(
+                                                                                                      const std::vector<std::string>& fileNames,
+                                                                                                      const std::string& treeName,
+                                                                                                      const std::string& branchName,
+                                                                                                      const bool onlyBestInMultibin,
+                                                                                                      const bool stripMatricesFromFurtherAttempts,
+                                                                                                      const bool onlyConvergedResults) {
```

Comment:
I would avoid changing the order here. This will break my code in lots of points.
But we can make default values for the later arguments as well. e.g.
`onlyBestInMultibin = false`
`stripMatricesFromFurtherAttempts = false`
`onlyConvergedResults = true`